teacher.rb

```diff
@@ -0,0 +1,11 @@
+require './person'
+class Teacher < Person
+  def initialize(age, name, specilization)
+    super(age, name)
+    @specilization = specilization
+  end
```

Comment:
Same here, kindly include the `parent_permission` parameter in your `initialize` and `super` constructors and make sure that the default value for `name` is  "Unknown" and the default value for `parent_permission`  is set to "true".